utils/db_queries.py

```diff
@@ -51,52 +55,26 @@ def server_filter(server):
 
 
 def get_zone_description(zone_id, ip_address=None, db=None):
-    if ip_address is None:
-        ip_address = cfmedb().hostname
-
-    if db is None:
-        db = Db(hostname=ip_address)
-
-    zones = list(
-        db.session.query(db["zones"]).filter(
-            db["zones"].id == zone_id
-        )
-    )
-    if zones:
-        return zones[0].description
-    else:
-        return None
+    db = _db(db, ip_address)
+    zone = db.get_first_of("zones", id=zone_id)
+    if zone is not None:
+        return zone.description
```

Comment:
its still returning none, i think silent makes sense, @psav may know details that warrant a warning